keiko-sql/src/main/kotlin/com/netflix/spinnaker/q/sql/SqlQueue.kt

```diff
@@ -842,25 +844,62 @@ class SqlQueue(
     }
   }
 
-  private fun ackMessage(fingerprint: String) {
+  private fun ackMessage(fingerprint: String, message: Message) {
+    if (log.isDebugEnabled) {
```

Comment:
Pardon my ignorance, is this required? If debug logging isn't enabled `log.debug` won't do anything anyway right?

Reply:
If fingerprint is big, we'll still spend non-trivial compute resources building the string to log even though log.debug doesn't actually log it.